### PR TITLE
[BSR-272] Add a complete subscribed webhook response

### PR DESCRIPTION
**Pull request by unmultimedio**
Include the complete subscribed object to the Create and List responses. Improve some comments.

### CODE REVIEW

File: proto/buf/alpha/registry/v1alpha1/webhook.proto [hunk 1]
```diff
@@ -42,24 +44,26 @@ message CreateWebhookRequest {
...
   WEBHOOK_EVENT_UNSPECIFIED = 0;
-  // WEBHOOK_EVENT_REPOSITORY_PUSH is emitted whenever a successful buf push
-  // is completed for a specific repository.
+  // WEBHOOK_EVENT_REPOSITORY_PUSH is emitted whenever a successful buf push is
+  // completed for a specific repository.
   WEBHOOK_EVENT_REPOSITORY_PUSH = 1;
 }
 
-// CreateWebhookResponse is the proto repsonse representation
+// CreateWebhookResponse is the proto response representation
```
Comment (robbertvanginkel) [inline proto/buf/alpha/registry/v1alpha1/webhook.proto:56]: Unrelated to this pr, but in general we haven't documented the `Foo{Request,Response}` types as they're tightly coupled to one service, for which the method should really have the comment on what it does.

File: proto/buf/alpha/registry/v1alpha1/webhook.proto [hunk 2]
```diff
@@ -42,24 +44,26 @@ message CreateWebhookRequest {
...
 // of a webhook request.
 message CreateWebhookResponse {
-  // For the user to unsubscribe from the webhook event.
-  string webhook_subscription_id = 1;
+  // Created webhook subscription.
+  SubscribedWebhook webhook = 1;
```
Comment (robbertvanginkel) [inline proto/buf/alpha/registry/v1alpha1/webhook.proto:60]: Could we potentially rename the `SubscribedWebhook` message to just `Webhook`? Its refferred to as webhook in all fields and comments now, feels a little off. The only thing you'd create or delete or list would be subscribed webhooks.

File: proto/buf/alpha/registry/v1alpha1/webhook.proto (hunk 1 above)
Comment (unmultimedio) [reply proto/buf/alpha/registry/v1alpha1/webhook.proto:56]: Done

File: proto/buf/alpha/registry/v1alpha1/webhook.proto [hunk 3]
```diff
@@ -81,16 +85,25 @@ message ListWebhooksRequest {
...
   // The event associated with the subscription id.
-  WebhookEvent webhook_event = 1;
+  WebhookEvent event = 1;
   // The id of the associated subscription.
-  string webhook_subscription_id = 2;
+  string webhook_id = 2;
+  // The webhook creation timestamp, immutable.
+  google.protobuf.Timestamp create_time = 3;
+  // The webhook last updated timestamp, mutable.
+  google.protobuf.Timestamp update_time = 4;
```
Comment (buildbreaker) [inline proto/buf/alpha/registry/v1alpha1/webhook.proto:102]: It feels that the mutability/immutability concept isn't really relevant in this API context

**Review by robbertvanginkel — APPROVED** [review]

**Review by buildbreaker — APPROVED** [review]

**Pull request merged**